### DISCUSSION TITLE
Data consumer - Find pictures of certain performances

### DISCUSSION THREAD

**Discussion by KlausIllmayer** [pair-cg/general-contributions#54] [User Stories]
# Motivation
I would like to give a surprise gift to a relative who used to be a theatre artist and create a photo book with images from the performances where this relative performed.

# 1. Need:

**Part 1: artifacts** 
To find pictures available online of Jane Doe performing in a theatre production between the years of 1990-2005 that are freely available to view/download digitally.

**Part 2: information related to the artifacts**
I also need to find basic production information about the productions whose pictures that Jane Doe is featured in.

# 2. Action:
* I use a search engine to find pictures of Jane Doe that feature her in a theatrical production where she is a part of the cast
* From the search results, I download all photographs so that I can create the photo book.
* I use a search engine to find basic information about the productions whose pictures were selected for the photo book. 

# 3. Desired Outcome:
Creating a photobook which is the best gift ever for my relative.

# 4. Perspective:
Data consumer

# 5. Stakeholder Group:
Data providers: arts organizations, archives, venues, artists, reviewers, media sites
Data curators

# 6. Additional info:
* The user story as described is in a lighter version (do you have a photograph of my relative from this performance?) one of the most requests to our performance database
* There is a lot of IPR (Intellectual property rights) involved in this user story that makes it nearly impossible to fulfill the need, but it could act as a good user story for building up data models that could support such a need.
* Often, there are photographs in reviews of a performance. Having this information relies either on a good data curation (and having time/money to do such a data curation) or could be done with the help of OCR and other text/image recognition strategies (but only if the original material is already digitized).
* This user story could act as a link between the user stories on federated search/data entity context and the archive user story

**Comment by fjjulien** [comment]
Many stakeholder groups have needs for pictures of performances. Presenting organizations who present touring productions have a direct relationship with the producing company and can usually obtain fairly easily (but "fairly easily" can be quite relative) quality photos with captions and full credits. Other stakeholder groups such as media professionals often have to place cold requests to obtain images for their articles or video news. As a researcher, I also need images to accompany my publications and I face the same challenges as media professionals: photos are hard to search, credits and caption are seldom available, and information about clearing the rights is never readily available.
